libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewOfflineRouter.java

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.services.android.navigation.ui.v5;
+
+import com.mapbox.services.android.navigation.v5.navigation.MapboxOfflineRouter;
+import com.mapbox.services.android.navigation.v5.navigation.NavigationRoute;
+import com.mapbox.services.android.navigation.v5.navigation.OfflineRoute;
+
+import timber.log.Timber;
+
+class NavigationViewOfflineRouter {
```

Comment:
We have a lot of objects called `routers`, since this seems to only take care of making sure that the offline router is properly configured, can we try to name it to make that apparent?

Reply:
Not sure I completely follow regarding sole responsibility of configuration, `findRouteWith` is also finding new offline routes

Reply:
Ok I understand what you are saying. Can we call this a `wrapper` or something? Since it is acting mostly as a wrapper for the routers.